driver/razercommon.h

```diff
@@ -49,6 +49,7 @@ do { \
 #define RED_PROFILE_LED   0x0C
 #define GREEN_PROFILE_LED 0x0D
 #define BLUE_PROFILE_LED  0x0E
+#define MAMBA_ELITE       0x0F
```

Comment:
0x0f is the `Docked LED`

Reply:
Yep. This device doesn't have any LED ID. I just set it to some random value to distinguish it during debugging. Will rewrite this part.